php/libraries/Utility.class.inc

```diff
@@ -849,33 +697,14 @@ class Utility
         return array();
     }
 
-
-    /**
-     * Return the number of columns to use before the quat table scripts
-     * overflow into a new table. This information comes from the config.xml.
-     *
-     * @return integer Column threshold from config file
-     * @note   Function should be either removed or moved to NDB_Config
-     * @note   Function comment written by Dave, who can't remember if he's
-     *       the author of this function.
-     *
-     * @cleanup
-     */
-    static function getColumnThresholdCount()
-    {
-        $config =& NDB_Config::singleton();
-        // get the list of Subprojects
-        $column_threshold = $config->getSetting('columnThreshold');
-        return $column_threshold;
-    }
-
     /**
      * Checks to see if a table contains a specified column
      *
      * @param string $table_name The table to check for a column.
      * @param string $column     The column name to check the table for.
      *
      * @return boolean true if the table has at least one NULL in the column
+     * @note   This should really be in the Database.class.inc file... -anyhowstep
```

Comment:
I left this note in .. @ridz1208 should it be moved to Database?

Reply:
probably, but not as part of this PR, and the note is still pointless to have. If anything, there should be a redmine ticket for it (or just a PR that does it and deprecates this)

Reply:
I'm not against the notes, i tend to look at code more then redmine myself